pod/perlintro.pod

```diff
@@ -584,10 +584,33 @@ the meantime, here's a quick cheat sheet:
  ^                   start of string
  $                   end of string
 
-Quantifiers can be used to specify how many of the previous thing you
-want to match on, where "thing" means either a literal character, one
-of the metacharacters listed above, or a group of characters or
-metacharacters in parentheses.
+Note that in the above, C<$> doesn't mean a dollar sign.  Similarly
```

Comment:
Perhaps slightly clearer to say, "`$` doesn't match a dollar sign."

Reply:
And if so, "mean" could be replaced with "match" to clarify some other statements in this section.

Reply:
Good points;  check out the changed version